scripts/apps/search/helpers.tsx

```diff
@@ -171,7 +172,9 @@ export function renderArea(
     }
 
     /* globals __SUPERDESK_CONFIG__: true */
-    const listConfig = itemProps.listConfig || appConfig.list || DEFAULT_LIST_CONFIG;
+    const listConfig = itemProps.view === 'compact-configurable'
```

Comment:
adjust this so if `itemProps.listConfig` is passed, it is used regardless of other options